src/components/VGrid/VGrid.tsx

```diff
@@ -214,7 +214,8 @@ export class VGrid<T, K extends keyof T> extends React.Component<Props<T, K>, St
     });
     if (!matched) throw new Error('No matched media');
     const { gridGap, minContentLength } = matched;
-    const repeatLength = minContentLength ? ~~((containerWidth + gridGap) / (minContentLength + gridGap)) : 1;
+    const width = minContentLength ? Math.max(minContentLength, containerWidth) : containerWidth;
+    const repeatLength = minContentLength ? ~~((width + gridGap) / (minContentLength + gridGap)) : 1;
```

Comment:
`repeatLength` is shoud be always greater than 1. So, `containerWidth` must not be less than the value of` minContentLength` :pray: